test/unit/lib/aws/request.test.js

```diff
@@ -152,6 +152,23 @@ describe('#request', () => {
     return expect(res).to.eql({ region: 'ap-northeast-1' });
   });
 
+  it('should invoke expected ECR methods', async () => {
```

Comment:
@pgrzesik this looks like a related test case so I dropped a new test here. What do you think?

Reply:
Hey @wwwehr - thank you. I think it would be better to test it as a part of this test suite: https://github.com/serverless/serverless/blob/95e063d904443044b73d61e73d04447b298d22ed/test/unit/lib/plugins/aws/provider.test.js#L1120

Here we are mocking the `ECR.createRepository`, so I think the best way would be to prepare a similar test case to the ones that are already there but one that will set `scanOnPush` property and will confirm that the `createRepository` was called with expected property. What do you think?

Reply:
Yep, I agree. Okay, I'll get it done. Cheers

Reply:
I think we can safely remove this test